website/docs/components/button/partials/accessibility/accessibility.md

```diff
@@ -1,6 +1,10 @@
+## Conformance rating
+
+<Doc::Badge @type="success">Conformant</Doc::Badge>
+
 This component has been designed and implemented with accessibility in mind. When used as recommended, there should not be any WCAG conformance issues with this component.
```

Comment:
I removed the first sentence because it's better communicated as a core principle than as a part of the component documentation for every component.

```suggestion
When used as recommended, there should not be any WCAG conformance issues with this component.
```